### ISSUE TITLE
Can't launch Spyder before installing Anaconda

### ISSUE THREAD

**Issue by anellox** [spyder-ide/spyder#15121]
### Issue Report Checklist

* [ x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [ x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x ] Tried basic troubleshooting (if a bug/error)
    * [ x] Restarted Spyder
    * [x ] Reset preferences with ``spyder --reset``
    * [x ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x ] Tried the other applicable steps from the Troubleshooting Guide
* [x ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

First of all I had installed spyder with no program and it worked fine, but now before I installed Ananconda it dindt open anymore.

 I've tried to uninstall spyder and anaconda and install everything again but it didn't work for me.

### Paste Traceback/Error Below (if applicable)

```python-traceback
Update LANGUAGE_CODES (inside config/base.py) if a new translation has been added to Spyder
Traceback (most recent call last):
  File "/home/warley/anaconda3/bin/spyder", line 11, in <module>
    sys.exit(main())
  File "/home/warley/anaconda3/lib/python3.8/site-packages/spyder/app/start.py", line 210, in main
    from spyder.app import mainwindow
  File "/home/warley/anaconda3/lib/python3.8/site-packages/spyder/app/mainwindow.py", line 72, in <module>
    from spyder.app import tour
  File "/home/warley/anaconda3/lib/python3.8/site-packages/spyder/app/tour.py", line 37, in <module>
    from spyder.utils.qthelpers import add_actions, create_action
  File "/home/warley/anaconda3/lib/python3.8/site-packages/spyder/utils/qthelpers.py", line 34, in <module>
    from spyder.utils.icon_manager import ima
  File "/home/warley/anaconda3/lib/python3.8/site-packages/spyder/utils/icon_manager.py", line 23, in <module>
    from spyder.utils.palette import QStylePalette, SpyderPalette
  File "/home/warley/anaconda3/lib/python3.8/site-packages/spyder/utils/palette.py", line 12, in <module>
    from qdarkstyle.colorsystem import Blue, Gray
ModuleNotFoundError: No module named 'qdarkstyle.colorsystem'

```

## Versions

* Spyder version: 5.0.0
* Python version: 3.8.8
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version: Linux Mint 20
cabuloso

_Thank you Guys soo much!_

**Comment by steff456** [comment]
Hi @anellox,

This is a duplicate of #15113, we are trying to fix this issue as fast as we can. For now please install spyder from the `conda-forge` channel and not from defaults.

Thanks for reporting!
